packages/kas/src/nodes.ts

```diff
@@ -3485,19 +3612,15 @@ export class Float extends Num {
     }
 }
 
-Num.Neg = new Int(-1).addHint("negate");
-Num.Sub = new Int(-1).addHint("subtract");
-Num.Div = new Int(-1).addHint("divide");
+const NumNeg = new Int(-1).addHint("negate");
+const NumSub = new Int(-1).addHint("subtract");
+const NumDiv = new Int(-1).addHint("divide");
 
-Num.Sqrt = new Rational(1, 2).addHint("root");
+const NumSqrt = new Rational(1, 2).addHint("root");
 
-Num.Zero = new Int(0);
-Num.One = new Int(1);
-Num.Ten = new Int(10);
-
-// set identities here
-Add.prototype.identity = Num.Zero;
-Mul.prototype.identity = Num.One;
+const NumZero = new Int(0);
+const NumOne = new Int(1);
+const NumTen = new Int(10);
```

Comment:
This can't be added to the `Num` class as static properties because `Int` and `Rational` depend on `Num` being defined.  We could probably drop the `Num` prefix.  Then we could export `Zero` and `One` directly here instead of down below.